mrmustard/physics/representations.py

```diff
@@ -310,3 +308,143 @@ def plot(
         ax.set_title(f"${title}$")
         plt.show(block=False)
         return fig, ax
+
+class Fock(Representation):
+    r"""This class is the Fock representation of a broad class of quantum states,
+    transformations, measurements, channels, etc.
+
+    The ansatz available in this representation is the multidimensional Tensor.
```

Comment:
Maybe use the ansatz's name? 
"The ansatz available in this representation is the ``ArrayAnsatz``."